console/src/main/java/org/eclipse/kapua/app/console/client/group/GroupEditDialog.java

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *
+ *******************************************************************************/
+package org.eclipse.kapua.app.console.client.group;
+
+import org.eclipse.kapua.app.console.shared.model.GwtGroup;
+import org.eclipse.kapua.app.console.shared.model.GwtSession;
+import org.eclipse.kapua.app.console.shared.service.GwtGroupService;
+import org.eclipse.kapua.app.console.shared.service.GwtGroupServiceAsync;
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.user.client.rpc.AsyncCallback;
+
+public class GroupEditDialog extends GroupAddDialog {
+
+    private final static GwtGroupServiceAsync gwtGroupService = GWT.create(GwtGroupService.class);
+    private GwtGroup selectedGroup;
+
+    public GroupEditDialog(GwtSession currentSession, GwtGroup selectedGroup) {
+        super(currentSession);
+        this.selectedGroup = selectedGroup;
+    }
+
+    @Override
+    public void createBody() {
+
+        super.createBody();
+        load();
+    }
+
+    @Override
+    public void submit() {
+        selectedGroup.setGroupName(groupNameField.getValue());
+        gwtGroupService.update(selectedGroup, new AsyncCallback<GwtGroup>() {
+
+            @Override
+            public void onFailure(Throwable arg0) {
+                m_exitStatus = false;
+
+            }
+
+            @Override
+            public void onSuccess(GwtGroup arg0) {
+                m_exitStatus = true;
+
+                hide();
```

Comment:
There should be a message to be shown on the bottom right corner about the result of the operation